cmake/Modules/FindITT.cmake

```diff
@@ -0,0 +1,28 @@
+# - Try to find ITT
+#
+# The following are set after configuration is done:
+#  ITT_FOUND          : set to true if ITT is found.
+#  ITT_INCLUDE_DIR    : path to ITT include dir.
+#  ITT_LIBRARIES      : list of libraries for ITT
+
+IF (NOT ITT_FOUND)
+SET(ITT_FOUND OFF)
+
+SET(ITT_INCLUDE_DIR)
+SET(ITT_LIBRARIES)
+
+IF(NOT CMAKE_BUILD_TYPE)
+  SET(CMAKE_BUILD_TYPE "Release" CACHE STRING
+      "Choose the type of build, options are: Debug Release"
+      FORCE)
+ENDIF(NOT CMAKE_BUILD_TYPE)
```

Comment:
Can you explain why this is needed?
```suggestion
```

Reply:
This is to build ITTAPI into release mode.

Reply:
But it should be build in the same mode as rest of the build, isn't it?

Reply:
Yes, it follows the rest of the build.